File: binary.pl
#!/usr/bin/perl


use strict;
use warnings;
use diagnostics;

use 5.10.0;

for my $n (1..100) {
    # Print word if...
    !($n % 15) ?    say "FizzBuzz"    :   
    !($n % 3)  ?    say "Fizz"        :   
    !($n % 5)  ?    say "Buzz"        :   
                    say "$n";            
}

File: binary.java
public class main {
	public static void main(String[] arg) {
		
		for(int i = 1; i < 100; i++) {
			if(i % 5 == 0 && i % 7 == 0)
				System.out.println("FizzBuzz");
			else if(i % 5 == 0)
				System.out.println("Fizz");
			else if(i % 7 == 0)
				System.out.println("Buzz");
			else
				System.out.println(i);
		}
	}
}
